Sources_v3/Controllers/DataController.swift

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// The base class for all controllers. Not meant to be used directly.
+public class DataController: Controller {
```

Comment:
I was going to comment "I feel DataController is too generic..." but then I realized I don't have a better suggestion. `RemoteDataController` doesn't work since this also deals with local data.... Oh well...

Reply:
@VojtaStavik my comment was towards the name of the entity, not the functionality

Reply:
I understand. What I wanted to say is that it's fine the name is too generic because it's a "generic" base class. You can say the same thing for `Controller`, it's even worse :)